Readme.md

```diff
@@ -25,9 +56,19 @@ render backends, it focuses only on the actual UI.
 - No global or hidden state
 - Customizable library modules (you can compile and use only what you need)
 - Optional font baker and vertex buffer output
-- [Documentation](https://Immediate-Mode-UI.github.io/Nuklear/doc/nuklear.html)
+- [Documentation](https://github.com/Immediate-Mode-UI/Nuklear/wiki)
+
+
+## Quick start 🚀
+1. Download the [header](https://raw.githubusercontent.com/Immediate-Mode-UI/Nuklear/master/nuklear.h) and place it in your project
+2. Include `nuklear.h` and define `NK_IMPLEMENTATION`
+```c
+#define NK_IMPLEMENTATION
+#include "nuklear.h"
+```
+3. Take a look at one of the [backend](https://github.com/Immediate-Mode-UI/Nuklear/tree/master/demo) examples to see how you can plug Nuklear in your backend of choice (Or just copy paste the files for your preferred backend into your project to get up and running fast💯) . See [Getting Started]() for in depth steps and examples on setting up Nuklear with a rendering backing.
```

Comment:
Not sure what the :100: is supposed to mean here :) I don't mind the :rocket: in the header, but the :100: seems random at best

Reply:
Thanks for taking a look. It is very random indeed. The 100 emoji seems to somehow slip in every now and then...